spec/spec_helper.rb

```diff
@@ -1,2 +1,5 @@
 require 'summon/refworks'
-require 'summon/transport/canned'
\ No newline at end of file
+require 'summon/ris'
+require 'summon/endnote'
+require 'summon/transport/canned'
+require 'byebug'
```

Comment:
I don't know that we want to leave `byebug` in here, is it really required for testing or only when we need it?